lib/json_web_token.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'jwt'
+
+##
+# Module for encoding and decoding JWTs.
+module JsonWebToken
+  def self.encode(payload, expiration = 30.minutes.from_now)
+    payload[:exp] = expiration.to_i
+    JWT.encode(payload, Rails.application.credentials.secret_key_base)
+  end
+
+  def self.decode(token)
+    decoded_token = JWT.decode(token, Rails.application.credentials.secret_key_base)[0]
```

Comment:
you know... from my PR too, I was thinking about how to elegantly pop this value, since `.first` seems so silly. I guess in the future we can do something like:

`(payload, _) = JWT.decode...`

And that'd more elegantly describe the response (this is not a suggested change for you just a thought I had looking over this)

Reply:
Nice, I like that a lot. I made the change and it works, so yeah it might be something we could apply on vets-api.